src/main/java/com/checkmarx/flow/service/SastScanner.java

```diff
@@ -226,17 +223,12 @@ public ScanResults cxFullScan(ScanRequest request, String path) throws ExitThrow
             String effectiveProjectName = projectNameGenerator.determineProjectName(request);
             request.setProject(effectiveProjectName);
 
-            String cxZipFile = FileSystems.getDefault().getPath("cx.".concat(UUID.randomUUID().toString()).concat(".zip")).toAbsolutePath().toString();
```

Comment:
I think we should rename the method to be more clear with method purpose

Reply:
Done.